src/olympia/conf/prod/settings.py

```diff
@@ -101,9 +101,20 @@
         'scope': 'profile',
         'skip_register_redirect': True,
     },
+    'code-manager': {
+        'client_id': env('CODE_MANAGER_FXA_CLIENT_ID'),
+        'client_secret': env('CODE_MANAGER_FXA_CLIENT_SECRET'),
+        'content_host': 'https://accounts.firefox.com',
+        'oauth_host': 'https://oauth.accounts.firefox.com/v1',
+        'profile_host': 'https://profile.accounts.firefox.com/v1',
+        'redirect_url':
+            'https://addons.mozilla.org/api/v4/accounts/authenticate/',
+        'scope': 'profile',
+        'skip_register_redirect': True,
+    },
```

Comment:
Are prod and stage ready to be enabled yet?

Reply:
no, not yet but the config can still be added, no?